pipelines/dax/docker/build_imgserv.groovy

```diff
@@ -13,49 +13,34 @@ node('jenkins-master') {
 }
 
 notify.wrap {
-  def retries = 3
   def repo = 'webserv/imgserv'
   def tag = 'dax_latest'
 
-  def run = {
-    dir('imgserv/lsst-dm-ci') {
+  node('docker') {
+    timeout(time: 1, unit: 'HOURS') {
+        git([
```

Comment:
It looks like there are mixed 2-space and 4-space indents.

Reply:
Fixed.